temel_bilgiler/giriş/web-geliştirmeye-giriş.md

```diff
@@ -0,0 +1,97 @@
+### Giriş
+
+Web geliştiricinin görevi nedir? Bir bakıma, websitesi yapma ve onun bakımından sorumludurlar.
+
+Web geliştiriciler genelde ürününü veya hizmetini web'e taşımaya çalışan müşteriler için çalışırlar. Çalışma genelde proje odaklıdır ve müşterinin ihtiyaçlarını son ürüne koordine eden bir ekip ile işbirliği yapmayı içerir. Müşteri bir teknoloji şirketi, bir organizasyon veya bir hükümet olabilir. Çalışma Frontend, Backend veya FullStack web geliştirme içerebilir.
+
+Web geliştirme eğer mantıksal problemleri çözmeyi seviyor, yeni kullanışlı şeyler yapmayı seviyor ve yeni teknolojileri denemeyi seviyorsanız sizin için iyi bir meslek olabilir. Web geliştiriciler genelde iyi bir iş/yaşam dengesine sahiptir, diğer iş alanlarına göre daha dolgun maaş alırlar ve yüksek talep görürler. Web geliştirme iş fırsatlarınızı daha iyi anlamak için özel konumunuzu Google'da aratın.
+
+Web geliştirme hakkında daha fazla bilgi için, Wikipedia da [web dizayn a giriş](https://en.wikipedia.org/wiki/Web_design)'e bakabilirsiniz .
```

Comment:
Daha fazla bilgi için web tasarımını ele alan [bu blog yazısını](https://tr.wix.com/blog/makale/web-tasarim-nedir) okuyabilirsiniz.